.github/workflows/ingest-test-fixtures-update-pr.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           sudo apt-get install -y libmagic-dev poppler-utils libreoffice pandoc
           sudo add-apt-repository -y ppa:alex-p/tesseract-ocr5
           sudo apt-get install -y tesseract-ocr
+          sudo apt-get install -y tesseract-ocr-kor
```

Comment:
IIUC this runs before generating these types of PRs: https://github.com/Unstructured-IO/unstructured/pull/776

For education, what triggers these PRs?

Reply:
> IIUC this runs before generating these types of PRs: https://github.com/Unstructured-IO/unstructured/pull/776

Yes, that's correct.

> For education, what triggers those PRs?

Great question! So [here](https://github.com/Unstructured-IO/unstructured/blob/44411ecc59eddbc5d3859800e91bfa43bb25c1ae/.github/workflows/ingest-test-fixtures-update-pr.yml#L5) we trigger on: `workflow_dispatch`. More info on workflow_dispatch [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch), but tldr; it will add a trigger-able button in the Actions menu in Github, where you can manually launch the workflow.